lib/runtime/outline.js

```diff
@@ -60,14 +60,18 @@ export function activate (ink) {
       pane.setItems(outline)
     }, 300)))
 
-    updateEditor(ed).then(outlineItems => {
+    updateEditor(ed, {
+      noUpdateSymbols: true
```

Comment:
Nitpick, but I feel like `updateSymols: false` would be easier to parse (here and in the Julia code).